tools/hello-world

```diff
@@ -0,0 +1,489 @@
+#!/usr/bin/env bash
+
+# This script is meant as a hands-on, super-fast, show-me-the-microVM
+# introduction to Firecracker.
+#
+# It will download the latest Firecracker release binary, along with a sample
+# pair of kernel and rootfs files, then use it all to spawn a demo microVM.
+#
+# Usage:
+#  ./hello-world
+#
+# No other Firecracker files are required in order to run this script. In fact,
+# the only dependecies are:
+# - BASH (obviously)
+# - curl (for downloading the binary and the microVM image)
+# - /dev/kvm (Firecracker dependency)
+# - Linux x86_64 4.14+ (Firecracker dependency)
+# - awscli (only needed before the GitHub repo goes public, for acccessing the
```

Comment:
So, can be removed now?

Reply:
Yes, `awscli` is no longer needed. I had forgotten to remove this comment as well.